File: app/src/main/java/com/example/calculator/Model/History.java
package com.example.calculator.Model;

public class History {
    String answer;
    String expr;

    public History(String expr2, String answer2) {
        this.expr = expr2;
        this.answer = answer2;
    }

    public String getExpr() {
        return this.expr;
    }

    public void setExpr(String expr2) {
        this.expr = expr2;
    }

    public String getAnswer() {
        return this.answer;
    }

    public void setAnswer(String answer2) {
        this.answer = answer2;
    }
}

File: app/src/main/java/com/inator/calculator/model/History.kt
package com.inator.calculator.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity
class History(
    @ColumnInfo(name = "Expression")
    var expr: String,

    @ColumnInfo(name = "Answer")
    var answer: String,

    @ColumnInfo(name = "Date")
    var date: String,
) {
    @PrimaryKey(autoGenerate = true)
    var eid = 0
}